doc/.pre_processing.py

```diff
@@ -287,11 +307,23 @@ class Colors:
 with open('./api_reference/changelog.rst', mode='w+',  encoding='utf-8') as changelogfile:
   changelogfile.write('Changelog\n')
   changelogfile.write('=========\n')
-  actualVersion = 0.0
+  actual_version = 0.0
+  add_experimental_header = False
   
   for change in changelog:
-    if actualVersion != change[0]:
-      actualVersion = change[0]
+    if change[1] == 3:
+      if not add_experimental_header:
```

Comment:
No the logic here becomes clear.

Please set `experimental` to True in the beginning, remove the `not` from the condition and set `experimental` to False afterwards.